index.html

```diff
@@ -192,6 +192,33 @@ <h3 class="head">🔸Skill</h3>
                         
                     </div>
                 </section>
-               
+           <section class="section_3">
+            <h2 class="description-2">I'm always interested in hearing about new projects,
+                       so if you'd like to chat please get in touch.</h2>
+                       <form action="">
+                      <label for="name-input">
+                        <input type="text" name="name-input" id="name-input" maxlength = "30" placeholder="    Full name" required>
+                      </label><br>
+                      <label for="email-input">
+                        <input type="email" name="email-input" id="email-input" placeholder="    Email address" required>
+                      </label><br>
+                      <label for="more-text">
+                        <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here"></textarea required>
+                      </label>
+                      <button type="submit" class="submit">Get in touch</button>
```

Comment:
- [ ] You did a good job styling all the buttons inside the document :+1: :100:, but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md) you should add `:hover` and `:active` interactions to the design of your project because it will improve the user experience :+1: 

**Figma design**
![Screenshot from 2023-01-17 19-36-46](https://user-images.githubusercontent.com/105117832/213051091-013f904b-c440-4d4b-b6c1-bda3ea3a24ca.png)